data/CMakeLists.txt

```diff
@@ -4,4 +4,5 @@
 
 add_subdirectory(yarpmanager)
 add_subdirectory(yarpdataplayer)
+add_subdirectory(harness_dev)
```

Comment:
@drdanz I have extended `YARP_DATA_DIRS` in `harness.cpp` as you suggested me, adding a new folder inside `tests` called `conf` with the configuration files required by `TestFrameGrabberTest`.
All the changes have been squashed in e88c130 rebasing over `origin/devel`.
Is it ok for you ?